### PR TITLE
client js css and html connected

### PR DESCRIPTION
**Pull request by zahollingsworth1**

### CODE REVIEW

File: config/config.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "development": {
     "username": "root",
     "password": "default",
-    "database": "statues_db",
```
Comment (GloGuldager) [inline config/config.json:5]: Zach. I believe the name of the database is simply    statue

File: public/index.html [hunk 2]
```diff
@@ -4,8 +4,8 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <link href="../public/styles/styles.css" rel="stylesheet"/> 
-    <script src="../public/js/index.js"> </script>
+    <link href="./styles/styles.css" rel="stylesheet"/>
```
Comment (GloGuldager) [inline public/index.html:7]: Is this a duplicate styles.css link?

File: public/index.html [hunk 3]
```diff
@@ -22,6 +22,6 @@ <h3 class="title">Please enter your name and click the button to begin</h3>
             </form>
 
         </section>
-
+        <script src="./js/index.js"> </script>
```
Comment (GloGuldager) [inline public/index.html:25]: Is this a duplicate script src?

**Review by GloGuldager — COMMENTED** [review]
zach, can you do a git pull and merge with git master and redo the git push? It seems it is duplicating everything. Travis has flagged it as failing review.

**Review by BPresseisen — APPROVED** [review]

**Pull request merged**